bin/DashboardNotifier.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+from __future__ import print_function
+from __future__ import division
+import stomp
+import json
+import traceback
+import os
+import datetime
+import logging 
+from multiprocessing import Process
+
+def produce(file_path, logging, conn):
+    """
+    """
+    json_data=open(file_path)
+    message = json.load(json_data)
+    logging.debug("Producing...%s" % message)
+    try:
+        messageDict = json.dumps(message)
+        conn.send(messageDict, destination=authParams['MSG_QUEUE'] )
+    except Exception as ex:
+        msg = "Error contacting Message Broker"
+        msg += str(ex)
+        msg += str(traceback.format_exc())
+        logging.debug(msg)
+
+logging.basicConfig(filename='/data/srv/asyncstageout/current/config/log', level=logging.DEBUG)
+amq_auth_file = "/data/srv/asyncstageout/current/config/asyncstageout/amq_auth_file.json"
+opened = False
+
+try:
+    f = open(amq_auth_file)
+    authParams = json.loads(f.read())
+    opened = True
+    f.close()
+except Exception as ex:
+    msg = "Error loading auth params"
+    msg += str(ex)
+    msg += str(traceback.format_exc())
+    logging.debug(msg)
+
+try:
+    host = [(authParams['MSG_HOST'], authParams['MSG_PORT'])]
+    conn = stomp.Connection(host, authParams['MSG_USER'], authParams['MSG_PWD'])
+    conn.start()
+    conn.connect()
+except Exception as ex:
+    msg = "Error contacting Message Broker"
+    msg += str(ex)
+    msg += str(traceback.format_exc())
+    logging.debug(msg)
```

Comment:
I guess if you get an exception you do not ant to continue with the code below..